tasks/less.js

```diff
@@ -192,10 +193,14 @@ module.exports = function(grunt) {
 
   var parseVariableOptions = function(options) {
     var pairs = _.toPairs(options);
-    var output = '';
-    pairs.forEach(function(pair) {
-      output += '@' + pair[0] + ':' + pair[1] + ';';
-    });
+    var output = _.reduce(pairs, function(accumulator, pair) {
+      var variableName = pair[0];
+      var variableValue = pair[1];
+      if (typeof variableValue !== 'undefined' && variableValue !== null) {
```

Comment:
Maybe just check for `variableValue`?

Reply:
This is this way because, time ago you asked me to do this (read your old comment please):

> XhmikosR on 12 Mar 2017
> Can tyou change this to typeof !== 'undefined'? Also, is the check for null needed here?

I'll update it anyway.